geonode/maps/urls.py

```diff
@@ -36,11 +36,15 @@
 if check_ogc_backend(geoserver.BACKEND_PACKAGE):
     new_map_view = 'new_map'
     existing_map_view = 'map_view'
+    # TODO qlr for geoserver
+    map_download_qlr = 'map_download_qlr'
```

Comment:
hi @gustry i guess what you meant by hiding the link on geoserver side is to remove this line. am i right?

Reply:
Answered below